### PR TITLE
Support section

### PR DESCRIPTION
**Pull request by dolbilov**

### CODE REVIEW

File: pages/index.css [hunk 1]
```diff
@@ -19,3 +19,48 @@
...
 @import url(../blocks/button/_theme/button_theme_secondary.css);
 @import url(../blocks/button/_theme/button_theme_secondary-black.css);
+
+/* Support section */
+@import url(../blocks/support/support.css);
+@import url(../blocks/support/__heading/support__heading.css);
+@import url(../blocks/support/__container/support__container.css);
+@import url(../blocks/support/__contributors-heading/support__contributors-heading.css);
+@import url(../blocks/support/__contributors-list/support__contributors-list.css);
+@import url(../blocks/support/__contributors/support__contributors.css);
```
Comment (fedbka) [inline pages/index.css:189]: Emty CSS-class

Comment (fedbka) [inline pages/index.css:189]: Empty CSS-class `support__contributors`

File: blocks/donation-form/__radiobuttons-container/_for/donation-form__radiobuttons-container_for_amount.css [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  gap: 16px 5px;
+}
+
+@media screen and (max-width: 1439px) {
+  .donation-form__radiobuttons-container_for_amount {
+    column-gap: 16px;
+  }
+}
+
+@media screen and (max-width: 1439px) {
```
Comment (fedbka) [inline blocks/donation-form/__radiobuttons-container/_for/donation-form__radiobuttons-container_for_amount.css:12]: indetical `max-width` in media-queries

File: blocks/donation-form/__fieldset/_for/donation-form__fieldset_for_data.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  margin-top: 41px;
+}
+
+@media screen and (max-width: 1439px) {
+  .donation-form__fieldset_for_data {
+    margin-top: 48px;
+  }
+}
+
+@media screen and (max-width: 1439px) {
```
Comment (fedbka) [inline blocks/donation-form/__fieldset/_for/donation-form__fieldset_for_data.css]: indentical `max-width` in media-queries

File: blocks/donation-form/__fieldset/_for/donation-form__fieldset_for_method.css [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  margin-top: 24px;
+}
+
+@media screen and (max-width: 1439px) {
+  .donation-form__fieldset_for_method {
+    margin-top: 14px;
+  }
+}
+
+@media screen and (max-width: 1439px) {
```
Comment (fedbka) [inline blocks/donation-form/__fieldset/_for/donation-form__fieldset_for_method.css]: indentical 'max-width' in media-queries

File: blocks/donation-form/__legend/_for/donation-form__legend_for_method.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+.donation-form__legend_for_method {
+  margin-bottom: 16px;
+}
+
+@media screen and (max-width: 1439px) and (min-width: 768px) {
```
Comment (fedbka) [inline blocks/donation-form/__legend/_for/donation-form__legend_for_method.css:5]: observing the accepted practice, using two rules would be more practical

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_maestro.css [hunk 6]
```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_maestro {
```
Comment (fedbka) [inline blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_maestro.css:1]: modificator without block or element

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_mastercard.css [hunk 7]
```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_mastercard {
```
Comment (fedbka) [inline blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_mastercard.css:1]: modificator without block or element

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_mir.css [hunk 8]
```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_mir {
```
Comment (fedbka) [inline blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_mir.css:1]: modificator without block or element

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_umoney.css [hunk 9]
```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_umoney {
```
Comment (fedbka) [inline blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_umoney.css:1]: modificator without block or element

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_webmoney.css [hunk 10]
```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_webmoney {
```
Comment (fedbka) [inline blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_webmoney.css:1]: modificator without block or element

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_visa.css [hunk 11]
```diff
@@ -0,0 +1,4 @@
+.donation-form__payment-logo_type_visa {
```
Comment (fedbka) [inline blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_visa.css:1]: modificator without block or element

File: blocks/support/__contributor/support__contributor.css [hunk 12]
```diff
@@ -0,0 +1,16 @@
+.support__contributor {
+  display: grid;
+  grid-template: repeat(2, auto) / max-content 1fr max-content;
+}
+
+@media screen and (max-width: 1479px) {
```
Comment (fedbka) [inline blocks/support/__contributor/support__contributor.css]: incrorect `max-width' in media-query

**Review by fedbka — APPROVED** [review]
There are some moments, but in overall a great job

P.S. empty CSS-class `support__contributors`

File: blocks/donation-form/__legend/_for/donation-form__legend_for_method.css (hunk 5 above)
Comment (dolbilov) [reply blocks/donation-form/__legend/_for/donation-form__legend_for_method.css:5]: I guess it's a better solution than media query with recurring rule, isn't it?

File: blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_maestro.css (hunk 6 above)
Comment (dolbilov) [reply blocks/donation-form/__payment-logo/_type/donation-form__payment-logo_type_maestro.css:1]: There is no common rules for logos. And creating empty files is undesirable.

**Pull request merged**